.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
+  "PreviewInSolutionExplorer": false
+}
```

Comment:
this is not needed right?

Reply:
No, it isn't needed, sorry about that.

Reply:
all good. Let me know when you removed it, I will approve it. LGTM. Thank you for the change

Reply:
Bumped the version to 1.4.9 and deleted the needless file.